lib/offline/download_manager.js

```diff
@@ -85,11 +96,12 @@ shaka.offline.DownloadManager = class {
    *    group will be created.
    * @param {shaka.extern.Request} request
    * @param {number} estimatedByteLength
+   * @param {boolean} initSegment
```

Comment:
Since this is boolean, please give it a name like isInitSegment.  That way, a reader doesn't think it is the an init segment object or byte array.

Reply:
Done!